app/experimenter/experiments/tests/test_tasks.py

```diff
@@ -405,6 +405,15 @@ def test_accepted_experiment_becomes_live_if_normandy_enabled(self):
             ).exists()
         )
 
+    def test_launch_email_sent_when_experiment_becomes_live(self):
+        ExperimentFactory.create_with_status(
+            target_status=Experiment.STATUS_ACCEPTED, normandy_id=1234
+        )
+
+        tasks.update_experiment_status()
+
+        self.assertEqual(len(mail.outbox), 1)
```

Comment:
Couldn't you just fold this assertion in the test directly above?  It's testing basically the same thing I think?